superset/assets/javascripts/explore/components/ExploreViewContainer.jsx

```diff
@@ -101,13 +106,47 @@ class ExploreViewContainer extends React.Component {
     }
   }
 
+  addHistory({ isReplace = false, title }) {
+    const { payload } = getExploreUrlAndPayload({ formData: this.props.form_data });
+    const longUrl = getExploreLongUrl(this.props.form_data);
+    if (isReplace) {
+      history.replaceState(
+        payload,
+        document.title,
```

Comment:
should this be `title` instead of `document.title`? or make the `document.title = title;` call before these?

Reply:
should be `title`. fixed.